src/mappings/generatedColumns.ts

```diff
@@ -39,12 +39,12 @@ export const extractEventArgs = (args: any[]) => {
  * Function to get value for a specific key
  * It searches for snake_case and camelCase value for the given key
  */
-export const extractValue = (obj: unknown, key: string): string => {
+export const extractValue = <T>(obj: unknown, key: string): T => {
   if (obj) {
     if (Object.keys(obj).includes(key)) {
-      return obj[key];
+      return obj[key] as T;
     }
-    return obj[snakeToCamelCase(key)];
+    return obj[snakeToCamelCase(key)] as T;
```

Comment:
Just a comment.

I am not a fan of blind casting like this, but I don't have any suggestions to improve it. I guess if its wrong somewhere we will get stuck on a block and figure out how to fix it.

Reply:
I agree to that too. From wherever we would be using this method, we would beforehand know the type we want to extract and if not passed I think this works as any by default. So I guess we are safe on the casting side.